requirements.txt

```diff
@@ -6,7 +6,7 @@ flask-login
 flask_mail==0.9.1
 Flask-Session
 Flask-SQLAlchemy==2.3.1
-flask-user
+Flask-User<0.7
```

Comment:
please elaborate on this change.

Reply:
Newest version - Flask-user 1.0, is alpha release. Additionally it is incompatible with older version - Flask-user 0.6.*, and older release have slightly better community support ;D

Reply:
Are we using it in app anyway?

Reply:
Yes, in models/users.py